documentation/staging/content/managing-domains/domain-lifecycle/scaling.md

```diff
@@ -16,36 +16,77 @@ WebLogic Server supports two types of clustering configurations, configured and
+```shell
+$ kubectl patch cluster cluster-1 --type=merge -p '{"spec":{"replicas":3}}'
+  cluster.weblogic.oracle/cluster-1 patched
+ 
+$ kubectl get clusters cluster-1 -o jsonpath='{.spec.replicas}'
+  3
+```
+
+#### On-demand, updating the Cluster or Domain directly
+##### Updating the Cluster directly
+To scale an individual WebLogic cluster directly, one can simply edit the `replicas` field of its associated Cluster resource. This can be done by using `kubectl`. More specifically, you can modify the Cluster directly by using the `kubectl edit` command.  For example:
+```shell
+$ kubectl edit cluster cluster-1 -n [namespace]
+```
+Here we are editing a Cluster named `cluster-1`. The `kubectl edit` command will open the Cluster definition in an editor and allow you to modify the `replicas` value directly. Once committed, the operator will be notified of the change and will immediately attempt to scale the corresponding cluster by reconciling the number of running pods/Managed Server instances with the `replicas` value specification.
 ```yaml
 spec:
   ...
-  clusters:
-    - clusterName: cluster-1
-      replicas: 1
+  clusterName: cluster-1
+  replicas: 1
   ...
 ```
-
-Alternatively, you can specify a default `replicas` value for all the clusters.  If you do this, then you don't need to list the cluster in the Domain (unless you want to customize another property of the cluster).
+##### Updating the Domain directly
+To scale a WebLogic cluster, that does not reference a Cluster resource or the Cluster resource does not have the `replicas` field specified, modify the value of the `replicas` field of a Domain by using the `kubectl edit` command.  For example:
+```shell
+$ kubectl edit domain domain1 -n [namespace]
+```
+Here we are editing a Domain named `domain1`. The `kubectl edit` command will open the Domain definition in an editor and allow you to modify the `replicas` value directly. Once committed, the operator will be notified of the change and will immediately attempt to scale the corresponding cluster by reconciling the number of running pods/Managed Server instances with the `replicas` value specification.
 ```yaml
 spec:
   ...
   replicas: 1
   ...
 ```
-In addition, see the helper scripts in the [Domain lifecycle sample scripts]({{< relref "/managing-domains/domain-lifecycle/startup#domain-lifecycle-sample-scripts" >}}) section.
+Alternatively, you can specify a default `replicas` value for all the WebLogic clusters.  If you do this, then you don't need to list any Cluster resources in the Domain (unless you want to customize another property of the cluster).
+
+{{% notice note %}}
+The `domain.spec.replicas` field affects any WebLogic clusters that do not specify a reference to a corresponding Cluster resource in the Domain resource or the corresponding Cluster resource does not have the `replicas` field specified.
```

Comment:
Perhaps put the note at the top of the section? It sets the overall tone for this section. Plus slightly reword:

The `domain.spec.replicas` field is the default `replicas` for all clusters in a domain. It affects any WebLogic clusters that do not have a corresponding Cluster resource and any WebLogic clusters with a Cluster resource that do not have the `.spec.replicas` field specified.